bm_serial_messages.h

```diff
@@ -125,12 +131,86 @@ typedef struct {
 } __attribute__ ((packed)) bm_serial_dfu_finish_t;
 
 typedef struct {
-  // Node id 
+  // Node id
   uint64_t node_id;
   // Reboot Reason
   uint32_t reboot_reason;
-  // git hash 
+  // git hash
   uint32_t gitSHA;
-  // reboot count 
+  // reboot count
   uint32_t reboot_count;
 } __attribute__ ((packed)) bm_serial_reboot_info_t;
+
+typedef struct {
+  // Node ID of the target node for which the request is being made. (Zeroed = all nodes)
```

Comment:
just realized that this is not supported on Spotter, rn it will try to get the info for a node_id == 0. If this is a required behavior I can add that to spotter!